Contest/Assignment/src/org/togetherjava/event/elevator/models/Human.java

```diff
@@ -0,0 +1,190 @@
+    if (startingFloor <= 0 || destinationFloor <= 0) {
+      throw new IllegalArgumentException("Floors must be at least 1");
+    }
+    this.startingFloor = startingFloor;
+    this.destinationFloor = destinationFloor;
+    this.currentState = HumanState.IDLE;
+  }
+
+  /**
+   * Invoked once the elevator system is ready to accept requests. Transitions the human out of the
+   * IDLE state. If the starting and destination floors are identical, the human is marked as
+   * arrived immediately. Otherwise, the correct travel direction is determined, an elevator is
+   * requested at the starting floor, and the state changes to WAITING_FOR_ELEVATOR.
+   *
+   * @param floorPanelSystem panel system used to request an elevator
+   */
+  @Override
+  public void onElevatorSystemReady(FloorPanelSystem floorPanelSystem) {
+
+    final int start = this.startingFloor;
+
+    log.info("checking human at floor {}, state", start);
+
+    final int destination = this.destinationFloor;
+
+    if (start == destination) {
+
+      final HumanState state = HumanState.ARRIVED;
+
+      log.info("starting floor is the same as destination floor => marking state as {}", state);
+
+      this.currentState = state;
+
+      return;
+    }
+
+    TravelDirection direction = start < destination ? TravelDirection.UP : TravelDirection.DOWN;
+
+    log.info("computed direction for human at floor {} is {}", start, direction);
+
+    floorPanelSystem.requestElevator(start, direction);
+
+    HumanState newState = HumanState.WAITING_FOR_ELEVATOR;
+
+    log.info("updating human at floor {} state from {} to {}", start, this.currentState, newState);
+
+    this.currentState = newState;
+
+    log.info(" human at floor {} state updated", start);
+
+    log.info(
+        "human startin from floor {}, requesting elevator towards floor {}, direction {}",
+        start,
+        destination,
+        direction);
+  }
+
+  /**
+   * Handles the event when an elevator arrives at a floor.
+   *
+   * <p>If the human is currently waiting for an elevator and this event represents arrival at the
+   * human's current floor, the human can now enter the elevator and request their actual
+   * destination floor. The state has to change to TRAVELING_WITH_ELEVATOR.
+   *
+   * <p>If the human is currently traveling with this elevator and the event represents arrival at
+   * the human's destination floor, the human can now exit the elevator.
+   *
+   * @param elevatorPanel the panel of the elevator that arrived at a floor
+   */
+  @Override
+  public void onElevatorArrivedAtFloor(ElevatorPanel elevatorPanel) {
+    final int elevatorId = elevatorPanel.getId();
+    final int elevatorCurrentFloor = elevatorPanel.getCurrentFloor();
+
+    //    log.debug(
+    //        "human at floor {}, (state: {}) received elevator {} arrival at floor {}",
+    //        this.startingFloor,
+    //        this.currentState,
+    //        elevatorId,
+    //        elevatorCurrentFloor);
```

Comment:
dangling logs (also elsewhere in the codebase) instead of making proper use of logger levels